integration-cli/daemon/daemon.go

```diff
@@ -171,9 +173,14 @@ func (d *Daemon) getClientConfig() (*clientConfig, error) {
 	}, nil
 }
 
-// Start will start the daemon and return once it is ready to receive requests.
-// You can specify additional daemon flags.
-func (d *Daemon) Start(args ...string) error {
+// Start starts the daemon and return once it is ready to receive requests.Sto
```

Comment:
Perhaps the "You can specify additional daemon flags" should stay? (Same for `StartWithError` and `StartWithBusyBox`)